.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  'env': {
+    'browser': true,
+    'es2021': true,
+  },
+  'extends': [
+    'google',
+  ],
+  'parser': '@typescript-eslint/parser',
+  'parserOptions': {
+    'ecmaVersion': 'latest',
+    'sourceType': 'module',
+  },
+  'plugins': [
+    '@typescript-eslint',
+  ],
+  'rules': {
+  },
```

Comment:
you need to configure these two options as specified in step 6.3: https://github.com/holgiosalos/cypress-training-doc#6-an%C3%A1lisis-de-c%C3%B3digo-estatico

Reply:
Fixed